src/test/java/io/jenkins/plugins/analysis/core/columns/IssueCounterITest.java

```diff
@@ -0,0 +1,94 @@
+package io.jenkins.plugins.analysis.core.columns;
+
+import static io.jenkins.plugins.analysis.core.testutil.Assertions.assertThat;
+
+import io.jenkins.plugins.analysis.warnings.Java;
+import java.io.IOException;
+import java.util.List;
+
+import org.junit.Test;
+
+import com.gargoylesoftware.htmlunit.WebClient;
+import com.gargoylesoftware.htmlunit.html.DomNode;
+import com.gargoylesoftware.htmlunit.html.HtmlPage;
+
+import hudson.model.FreeStyleProject;
+import hudson.model.Result;
+import jenkins.model.Jenkins;
+
+import io.jenkins.plugins.analysis.core.testutil.IntegrationTestWithJenkinsPerTest;
+
+/**
+ * Integration Test for the Issue counter column.
+ */
+public class IssueCounterITest extends IntegrationTestWithJenkinsPerTest {
+
+    private static final String ISSUE_ELEMENT_XPATH = "/html/body/div[4]/div[2]/div[2]/div[2]/table/tbody/tr[2]/td[8]/a";
```

Comment:
Is this path stable? Wouldn't it be easier if we add a div with ID? I added one for your, see https://github.com/jenkinsci/warnings-ng-plugin/commit/bfab4b43725b97801eab4c9b2dc6724af048829b